src/ostorlab/serve_app/types.py

```diff
@@ -120,6 +128,15 @@ def resolve_detail(
                 ],
             )
 
+    def resolve_risk_rating(
+        self: models.Vulnerability, info: graphql_base.ResolveInfo
+    ) -> OxoRiskRatingEnum:
+        """Resolve risk rating of vulnerability"""
+        try:
+            return OxoRiskRatingEnum[self.risk_rating.name]
+        except KeyError:
+            return None
```

Comment:
Update the return type please! 

OxoRiskRatingEnum | None

Reply:
+1